CHANGELOG.md

```diff
@@ -0,0 +1 @@
+The change log is kept in [GitHub Releases](https://github.com/Homebrew/brew/releases).
```

Comment:
`See Homebrew's [releases on GitHub](https://github.com/Homebrew/brew/releases) for the changelog.`